specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -71,6 +83,8 @@ model AcsChatMessageEditedInThreadEventData
   editTime: utcDateTime;
 }
 
+//  const MicrosoftCommunicationChatMessageDeletedExample: AcsChatMessageDeletedEventData = #{deleteTime: "2021-02-19T00:43:10.14Z", messageId: "1613695388152", senderId: "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7d07-83fe-084822000f6e", senderCommunicationIdentifier: #{rawId: "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7d07-83fe-084822000f6e", communicationUser: #{id: "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7d07-83fe-084822000f6e"}}, senderDisplayName: "Bob(Admin)", composeTime: "2021-02-19T00:43:08.152Z", type: "Text", version: 1613695390361, recipientId: "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7d60-83fe-084822000f6f", recipientCommunicationIdentifier: #{rawId: "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7d60-83fe-084822000f6f", communicationUser: #{id: "8:acs:109f0644-b956-4cd9-87b1-71024f6e2f44_00000008-578d-7d60-83fe-084822000f6f"}}, transactionId: "fFs4InlBn0O/0WyhfQZVSQ.1.1.2.1.1867776045.1.4", threadId: "19:48899258eec941e7a281e03edc8f4964@thread.v2"};
```

Comment:
Why commented out?

Reply:
these examples do not match with the typespec so I left them in but commented them out